tests/js/plugin/enhancedanchoring-test.coffee

```diff
@@ -106,6 +110,17 @@ describe 'Annotator.Plugin.EnhancedAnchoring', ->
       assert.ok hl
       assert.equal hl.page, page
 
+    it 'announces the creation of the highlights in an event', ->
+      am = createAnchoringManager()
+
+      assert am.annotator.publish.notCalled
+
+      ann = createTestAnnotation "a1"
+      anchor = am.createAnchor(ann, ann.target[0]).result
+
+      hl = anchor.highlight[anchor.startPage]
+      assert am.annotator.publish.calledWith "highlightsCreated", [hl]
```

Comment:
On 2015-02-10 10:23, Aron Carroll wrote:
> Sinon provides it's own assertions
> <http://sinonjs.org/docs/#assertions> for spies that give more
> detailed output in the error messages.

OK, I migrated to using those.

> We usually mix them into the chai |assert| object at the top of the
> test file.

That mixing is magic.